src/runtime_src/core/tools/common/XBMain.h

```diff
@@ -19,13 +19,26 @@
 
 #include <string>
 #include <vector>
+#include <memory>
+
+// ------------ F O R W A R D - D E C L A R A T I O N S ----------------------
+// Forward declarations - use these instead whenever possible...
+class SubCmd;
   
-  typedef enum {
-    RC_SUCCESS = 0,
-    RC_ERROR_IN_COMMAND_LINE = 1,
-    RC_ERROR_UNHANDLED_EXCEPTION = 2
-  } ReturnCodes;
+// --------- E N U M E R A T I O N S   /   T Y P E D E F S--------------------
+typedef enum {
+  RC_SUCCESS = 0,

```

Comment:
Is RC_ too generic?  You are definitely not the first one to use RC_ (just google it).

Reply:
Removed the return type entirely.  We should be using exceptions instead and with my next set of submissions, I will update the code accordingly.